packages/frontend/src/helpers/variables.ts

```diff
@@ -35,28 +35,35 @@ function postProcess(
   metadata: IDataOutMetadata,
 ): Variable[] {
   const result: Variable[] = []
-
   for (const variable of variables) {
     const { name, ...rest } = variable
+
     // lodash get does its work by specifying the 'name' path
     const {
       isHidden = false,
       type = null,
-      label = null,
       order = null,
       displayedValue = null,
     } = get(metadata, name, {}) as IDataOutMetadatum
 
-    // the fields with .order will be hidden and have no output content (filtered later)
+    // only label may change if metadata exists
+    let { label = null } = get(metadata, name, {}) as IDataOutMetadatum
```

Comment:
nit: I think we don't need this extra `get` call. Instead, on line 61, do this:

```
result.push({
  // Default to showing lodash path if a label doesn't exist.
  label: label ?? name
```

Reply:
We don't need to worry about metadata not existing either; it's safe to display the lodash path as long as a label isn't provided.

Reply:
ok thanks for spotting!